src/certificates/data/api.js

```diff
@@ -28,37 +29,61 @@ export async function getCertificates(courseId) {
  * @param {object} certificatesData
  * @returns {Promise<Object>}
  */
-export async function createCertificate(courseId, certificatesData) {
-  const body = {
-    ...certificatesData,
-    description: 'Description of the certificate',
-    editing: true,
-    isActive: false,
-    name: 'Name of the certificate',
-    version: 1,
-    signatories: certificatesData.signatories
-      .map(signatory => convertObjectToSnakeCase(signatory, true))
-      .map(signatorySnakeCase => ({ ...signatorySnakeCase, certificate: null })),
-  };
 
+export async function createCertificate(courseId, certificatesData) {
   const { data } = await getAuthenticatedHttpClient()
     .post(
       getCertificateApiUrl(courseId),
-      convertObjectToSnakeCase(body, true),
+      prepareCertificatePayload(certificatesData),
     );
 
   return camelCaseObject(data);
 }
 
+/**
+ * Update course certificate.
+ * @param {string} courseId
+ * @param {object} certificateData
+ * @returns {Promise<Object>}
+ */
+export async function updateCertificate(courseId, certificateData) {
+  const { data } = await getAuthenticatedHttpClient()
+    .post(
+      getUpdateCertificateApiUrl(courseId, certificateData.id),
+      prepareCertificatePayload(certificateData),
+    );
+  return camelCaseObject(data);
+}
+
 /**
  * Delete course certificate.
  * @param {string} courseId
  * @param {object} certificateId
  * @returns {Promise<Object>}
  */
 export async function deleteCertificate(courseId, certificateId) {
-  const { data } = await getAuthenticatedHttpClient().delete(
-    getDeleteCertificateApiUrl(courseId, certificateId),
-  );
+  const { data } = await getAuthenticatedHttpClient()
+    .delete(
+      getUpdateCertificateApiUrl(courseId, certificateId),
+    );
   return data;
 }
+
+/**
+ * Activate/deactivate course certificate.
+ * @param {string} courseId
+ * @param {object} activationStatus
+ * @returns {Promise<Object>}
+ */
+export async function updateActivation(path, activationStatus) {
```

Comment:
```suggestion
export async function updateStatus(path, activationStatus) {
```
`Status` better? Update other cases

Reply:
What do you think about `updateActivationStatus`?